plugins/Microsoft.DocAsCode.Build.UniversalReference/uref.nuspec

```diff
@@ -9,7 +9,7 @@
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <licenseUrl>https://github.com/dotnet/docfx/blob/dev/LICENSE</licenseUrl>
     <projectUrl>https://github.com/dotnet/docfx</projectUrl>
-    <description>This is a plugin for DocFX to generate API documentation for projects in various languages. This package contains both plugin DLLs and templates.</description>
```

Comment:
Move into docfx instead?

Reply:
will do this in another PR

---
In reply to: [153388968](https://github.com/dotnet/docfx/pull/2234#discussion_r153388968) [](ancestors = 153388968)